### PR TITLE
[VEN-]: Addresses for zksync testnet and mainnet

### PR DESCRIPTION
**Pull request by GitGuru7**

### CODE REVIEW

File: deployed-contracts/governance.md [hunk 1]
```diff
@@ -145,4 +145,20 @@ Specific functions that can be invoked by the guardians
...
+
+## Zksync Sepolia
+
+* Access Control Manager: [`0xD07f543d47c3a8997D6079958308e981AC14CD01`](https://sepolia.explorer.zksync.io/address/0xD07f543d47c3a8997D6079958308e981AC14CD01)
+* Guardian: [`0xa2f83de95E9F28eD443132C331B6a9C9B7a9F866`](https://sepolia.explorer.zksync.io/address/0xa2f83de95E9F28eD443132C331B6a9C9B7a9F866)
+* Normal Timelock: [`0x1730527a0f0930269313D77A317361b42971a67E`](https://sepolia.explorer.zksync.io/address/0x1730527a0f0930269313D77A317361b42971a67E)
+* Fasttrack Timelock: [`0xb055e028b27d53a455a6c040a6952e44E9E615c4`](https://sepolia.explorer.zksync.io/address/0xb055e028b27d53a455a6c040a6952e44E9E615c4)
+* Critical Timelock: [`0x0E6138bE0FA1915efC73670a20A10EFd720a6Cc8`](https://sepolia.explorer.zksync.io/address/0x0E6138bE0FA1915efC73670a20A10EFd720a6Cc8)
+* Omnichain Governance Executor: [`0x83F79CfbaEee738173c0dfd866796743F4E25C9e`](https://sepolia.explorer.zksync.io/address/0x83F79CfbaEee738173c0dfd866796743F4E25C9e)
+* Omnichain Executor Owner Proxy : [`0xa34607D58146FA02aF5f920f29C3D916acAb0bC5`](https://sepolia.explorer.zksync.io/address/0xa34607D58146FA02aF5f920f29C3D916acAb0bC5)
```
Comment (chechu) [inline deployed-contracts/governance.md]:
```suggestion
* Omnichain Executor Owner Proxy: [`0xa34607D58146FA02aF5f920f29C3D916acAb0bC5`](https://sepolia.explorer.zksync.io/address/0xa34607D58146FA02aF5f920f29C3D916acAb0bC5)
```

File: risk/resilient-price-oracle.md [hunk 2]
```diff
@@ -177,6 +177,25 @@ The current list of correlated token oracles in Venus is:
...
 
+#### Zksync Sepolia
+| Pool | Market | MAIN oracle | PIVOT oracle | FALLBACK oracle | Notes |
+|---|---|---|---|---|---|
+| Core | WBTC | [ChainlinkOracle](https://sepolia.explorer.zksync.io/address/0x0DFf10dCdb3526010Df01ECc42076C25C27F8323) | - | - | |
+| Core | WETH | [ChainlinkOracle](https://sepolia.explorer.zksync.io/address/0x0DFf10dCdb3526010Df01ECc42076C25C27F8323) | - | - | |
+| Core | USDC_E | [ChainlinkOracle](https://sepolia.explorer.zksync.io/address/0x0DFf10dCdb3526010Df01ECc42076C25C27F8323) | - | - | |
+| Core | USDT | [ChainlinkOracle](https://sepolia.explorer.zksync.io/address/0x0DFf10dCdb3526010Df01ECc42076C25C27F8323) | - | - | |
+| Core | ZK | [ChainlinkOracle](https://sepolia.explorer.zksync.io/address/0x0DFf10dCdb3526010Df01ECc42076C25C27F8323) | - | - | |
+
```
Comment (chechu) [inline risk/resilient-price-oracle.md]: We are not adding the testnet configuration for the oracles. I would remove this section

File: deployed-contracts/isolated-pools.md [hunk 3]
```diff
@@ -429,3 +429,44 @@
...
+* Comptroller: [`0xC527DE08E43aeFD759F7c0e6aE85433923064669`](https://sepolia.explorer.zksync.io/address/0xC527DE08E43aeFD759F7c0e6aE85433923064669)
+* NativeTokenGateway: [`0xC2bc5881f2c1ee08a1f0fee65Fbf2BB4C4DD81e9`](https://sepolia.explorer.zksync.io/address/0xC2bc5881f2c1ee08a1f0fee65Fbf2BB4C4DD81e9)
+* Markets:
+  * vWBTC\_Core: [`0x9c2379ed8ab06B44328487f61873C7c44BD6E87D`](https://sepolia.explorer.zksync.io/address/0x9c2379ed8ab06B44328487f61873C7c44BD6E87D)
+  * vWETH\_Core: [`0x31eb7305f9fE281027028D0ba0d7f57ddA836d49`](https://sepolia.explorer.zksync.io/address/0x31eb7305f9fE281027028D0ba0d7f57ddA836d49)
+  * vUSDT\_Core: [`0x7Bfd185eF8380a72027bF65bFEEAb0242b147778`](https://sepolia.explorer.zksync.io/address/0x7Bfd185eF8380a72027bF65bFEEAb0242b147778)
+  * vUSDC_E\_Core: [`0x58b0b248BB11DCAA9336bBf8a81914201fD49461`](https://sepolia.explorer.zksync.io/address/0x58b0b248BB11DCAA9336bBf8a81914201fD49461)
+  * vZK\_Core: [`0x92f4BD794303c0BD0791B350Be5531DB38414f47`](https://sepolia.explorer.zksync.io/address/0x92f4BD794303c0BD0791B350Be5531DB38414f47)
+
+## Zksync Mainnet
```
Comment (chechu) [inline deployed-contracts/isolated-pools.md]: We are listing first mainnets and then testnets. So, this section should be moved up

File: deployed-contracts/oracles.md [hunk 4]
```diff
@@ -101,3 +101,19 @@
 * DefaultProxyAdmin:[`0xA78A1Df376c3CEeBC5Fab574fe6EdDbbF76fd03e`](https://sepolia.arbiscan.io/address/0xA78A1Df376c3CEeBC5Fab574fe6EdDbbF76fd03e)
 * OneJump Oracle weETH/eETH/USD (having intermediate oracle Chainlink): [`0x4f894aC6F5e2E2bD72632B137E28234339Db3D53`](https://sepolia.arbiscan.io/address/0x4f894aC6F5e2E2bD72632B137E28234339Db3D53)
 * OneJump Oracle wstETH/stETH/USD (having intermediate oracle Chainlink): [`0x06b47B32d9C4D8f948EB3FFDB4a1ecE93794f922`](https://sepolia.arbiscan.io/address/0x06b47B32d9C4D8f948EB3FFDB4a1ecE93794f922)
+
+## Zksync Sepolia
```
Comment (chechu) [inline deployed-contracts/oracles.md]: Switch the order: first zkSync mainnet, then zkSync sepolia

File: deployed-contracts/core-pool.md [hunk 5]
```diff
@@ -186,3 +186,17 @@
 * XVS Vault Proxy: [`0x407507DC2809D3aa31D54EcA3BEde5C5c4C8A17F`](https://sepolia.arbiscan.io/address/0x407507dc2809d3aa31d54eca3bede5c5c4c8a17f)
 * Prime: [`0xadb04ac4942683bc41e27d18234c8dc884786e89`](https://sepolia.arbiscan.io/address/0xadb04ac4942683bc41e27d18234c8dc884786e89)
 * Prime Liquidity Provider: [`0xe82c2c10f55d3268126c29ec813dc6f086904694`](https://sepolia.arbiscan.io/address/0xe82c2c10f55d3268126c29ec813dc6f086904694)
+
+## Zksync Sepolia
```
Comment (chechu) [inline deployed-contracts/core-pool.md]: Let's switch the order: first mainnet, then sepolia

File: deployed-contracts/governance.md [hunk 6]
```diff
@@ -145,4 +145,20 @@ Specific functions that can be invoked by the guardians
 * Fasttrack Timelock: [`0x14642991184F989F45505585Da52ca6A6a7dD4c8`](https://sepolia.arbiscan.io/address/0x14642991184F989F45505585Da52ca6A6a7dD4c8)
 * Critical Timelock: [`0x0b32Be083f7041608E023007e7802430396a2123`](https://sepolia.arbiscan.io/address/0x0b32Be083f7041608E023007e7802430396a2123)
 * Omnichain Governance Executor: [`0xcf3e6972a8e9c53D33b642a2592938944956f138`](https://sepolia.arbiscan.io/address/0xcf3e6972a8e9c53D33b642a2592938944956f138)
-* Omnichain Executor Owner Proxy : [`0xfCA70dd553b7dF6eB8F813CFEA6a9DD039448878`](https://sepolia.arbiscan.io/address/0xfCA70dd553b7dF6eB8F813CFEA6a9DD039448878)
\ No newline at end of file
+* Omnichain Executor Owner Proxy : [`0xfCA70dd553b7dF6eB8F813CFEA6a9DD039448878`](https://sepolia.arbiscan.io/address/0xfCA70dd553b7dF6eB8F813CFEA6a9DD039448878)
+
+## Zksync Sepolia
```
Comment (chechu) [inline deployed-contracts/governance.md]: Let's switch the order: first mainnet, then sepolia

File: deployed-contracts/xvs-multichain.md [hunk 7]
```diff
@@ -51,3 +51,15 @@
...
 * XVSProxyOFTDest: [`0xFdC5cEC63FD167DA46cF006585b30D03B104eFD4`](https://sepolia.arbiscan.io/address/0xFdC5cEC63FD167DA46cF006585b30D03B104eFD4)
 * XVS: [`0x877Dc896e7b13096D3827872e396927BbE704407`](https://sepolia.arbiscan.io/address/0x877Dc896e7b13096D3827872e396927BbE704407)
+
+### Zksync Sepolia
+
+* XVSBridgeAdmin: [`0x28cfE3f2D7D8944FAd162a058260ec922C19065E`](https://sepolia.explorer.zksync.io/address/0x28cfE3f2D7D8944FAd162a058260ec922C19065E)
+* XVSProxyOFTDest: [`0x760461ccB2508CAAa2ECe0c28af3a4707b853043`](https://sepolia.explorer.zksync.io/address/0x760461ccB2508CAAa2ECe0c28af3a4707b853043)
+* XVS: [`0x3AeCac43A2ebe5D8184e650403bf9F656F9D1cfA`](https://sepolia.explorer.zksync.io/address/0x3AeCac43A2ebe5D8184e650403bf9F656F9D1cfA)
+
+### Zksync Mainnet
```
Comment (chechu) [inline deployed-contracts/xvs-multichain.md]: This should be moved up, in the `Mainnet chains` section

**Review by chechu — COMMENTED** [review]
Let's use the term `ZKsync` instead of `Zksync`, as the project is doing in their website: https://zksync.io

File: deployed-contracts/governance.md (hunk 1 above)
Comment (GitGuru7) [reply deployed-contracts/governance.md]: [a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)

File: risk/resilient-price-oracle.md (hunk 2 above)
Comment (GitGuru7) [reply risk/resilient-price-oracle.md]: [a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)

File: deployed-contracts/isolated-pools.md (hunk 3 above)
Comment (GitGuru7) [reply deployed-contracts/isolated-pools.md]: [a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)

File: deployed-contracts/oracles.md (hunk 4 above)
Comment (GitGuru7) [reply deployed-contracts/oracles.md]: [a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)

File: deployed-contracts/core-pool.md (hunk 5 above)
Comment (GitGuru7) [reply deployed-contracts/core-pool.md]: [a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)

File: deployed-contracts/governance.md (hunk 6 above)
Comment (GitGuru7) [reply deployed-contracts/governance.md]: [a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)

File: deployed-contracts/xvs-multichain.md (hunk 7 above)
Comment (GitGuru7) [reply deployed-contracts/xvs-multichain.md]: [a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)

File: risk/resilient-price-oracle.md [hunk 8]
```diff
@@ -177,6 +177,16 @@ The current list of correlated token oracles in Venus is:
...
 
+#### ZKsync Mainnet
+| Pool | Market | MAIN oracle | PIVOT oracle | FALLBACK oracle | Notes |
+|---|---|---|---|---|---|
+| Core | WBTC | [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5) | - | - | |
+| Core | WETH | [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5) | - | - | |
+| Core | USDC_E | [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5) | - | - | |
+| Core | USDT | [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5) | - | - | |
+| Core | ZK | [RedStoneOracle](https://explorer.zksync.io/address/0xFa1e65e714CDfefDC9729130496AB5b5f3708fdA) | [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5)
+| [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5) | Upper bound: 1.01. Lower bound: 0.99 |
```
Comment (chechu) [inline risk/resilient-price-oracle.md]: There is an extra row

```suggestion
| Core | ZK | [RedStoneOracle](https://explorer.zksync.io/address/0xFa1e65e714CDfefDC9729130496AB5b5f3708fdA) | [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5) | [ChainlinkOracle](https://explorer.zksync.io/address/0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5) | Upper bound: 1.01. Lower bound: 0.99 |
```

**Pull request merged**